### DISCUSSION TITLE
Ability to map pause button.

### DISCUSSION THREAD

**Discussion by remclean** [ppy/osu#26980] [Ideas]
Currently, the only way to pause in-game is with esc. I can't do that, I have a keypad that I can't change the mappings on. There should be an option in the keybinds to change, or add, a different pause button.

**Comment by Joehuu** [comment]
Should be able to change this keybind to achieve the behavior you want or `Back`:

![osu!_bim3abL47G](https://github.com/ppy/osu/assets/35318437/e73cb9ec-ca2b-492a-9911-43b436045c46)

**Comment by remclean** [comment]
My bad, I still don't know how I missed it. I spent a solid 10 minutes looking for it too.
